moped-editor/src/views/projects/projectView/ProjectComponents/ComponentForm.js

```diff
@@ -180,11 +184,9 @@ const ComponentForm = ({
               id="signal"
               name="signal"
               control={control}
-              render={({ onChange, value, ref }) => (
+              render={({ field }) => (
                 <SignalComponentAutocomplete
-                  onChange={onChange}
-                  value={value}
-                  ref={ref}
+                  {...field}
```

Comment:
`onChange`, `value`, and `ref` are now packed into `field` which can be spread out into the component props.